### PR TITLE
Adds new property and method hasValue for Laminas\Form\Element

### LINKED ISSUE #63: FormRow: allow attributes for "valid" context
|    Q        |   A
|------------ | ------
| New Feature | yes
| RFC         | no
| BC Break    | maybe

Hi, FormRow view helper generates two outputs:

1. Element without errors
1. Element with errors

This schema lacks a third option, hidden in the first one; the outputs should be:

1. Element rendered, waiting to be filled
1. Element filled without errors
1. Element filled with errors

This is needed for proper styling, take for example the bootstrap ones (https://getbootstrap.com/docs/4.4/components/forms/#validation):

### Before submit

immagine

### After submit

immagine

With the current release, the valid fields cannot be styled to be green, as far as I can tell.

### PR DESCRIPTION
**Pull request by ALTAMASH80**
|    Q          |   A
|-------------- | ------
| Documentation | no
| BC Break      | no
| New Feature   | yes

### Description

This changed was agreed on the mentioned issue and I've just done the coding. Thanks!
https://github.com/laminas/laminas-form/issues/63#issuecomment-775355100

### CODE REVIEW

File: src/Element.php [hunk 1]
```diff
@@ -514,4 +522,13 @@ public function getMessages()
     {
         return $this->messages;
     }
+    
+    /**
+     * 
+     * @return boolean
+     */
+    public function hasValue()
```
Comment (Slamdunk) [inline src/Element.php:528]: This must be tested with a unit test at least

**Review by Slamdunk — CHANGES_REQUESTED** [review]
Please follow the `DCO` suggestion about the missing `sign-off`: click `Details` in the Checks box to see how to fix it

File: src/Element.php (hunk 1 above)
Comment (ALTAMASH80) [reply src/Element.php:528]: Ok, I've to write a test. I'll check an existing test and make one. Thanks for the suggestion.

**Comment by ALTAMASH80** [comment]
Hi @Slamdunk, now it says all checks have passed and branch appears in green. Thanks!

**Review by Slamdunk — CHANGES_REQUESTED** [review]
At the time of writing, Travis build is unavailable. Please run `composer check`, I'm sure it will fail: run `composer cs-fix` to fix the coding standards

**Comment by ALTAMASH80** [comment]
@Slamdunk, Thanks for sharing the code checking command line. That was a new thing to learn. Spaces are a problem, that is something. It reminded me of the days when underscore was a problem. The reason it uses up a byte, therefore, use Capital letters. In future properties will have m_bHasVal type of conventions? Thanks!

**Comment by Slamdunk** [comment]
Thank you @ALTAMASH80

**Pull request merged**